### PR TITLE
 [POKEMON-0003] Create WebServices Model

### PR DESCRIPTION
**Pull request by arriyam**
## Details

Create `WebServices` model.

## Related Tickets

Ticket: [POKEMON-0003](https://trello.com/c/nHo60qFd/2-implement-pokemon-data-retrieval-and-model)
Epic ticket: [POKEMON](https://trello.com/b/nsSxCgvo/pokemon)
To-do ticket : [POKEMON-0013](https://trello.com/c/dqgUd3R6/6-unit-tests)

## Motivation and Context

The model is used to create a correct URL to retrieve a random Pokemon from [PokéAPI](https://pokeapi.co/). Also, it contains a helper function needed to randomize Pokemon fetched.

### CODE REVIEW

**Comment by arriyam** [comment]
The following code won't work as it needs the code from the updated Pokemon Model [PR](https://github.com/arriyam/PokemonRampUp/pull/3). That is why I added the `DO NOT MERGE` label on this PR. Once the updated Pokemon Model [PR](https://github.com/arriyam/PokemonRampUp/pull/3) is merged in main I will merge main with this branch. Once that is done I can remove the `DO NOT MERGE` label and merge this branch to main. 

However, everything related to this PR is completed so it can be reviewed. Please take a look at it and let me know if there are any concerns.

**Comment by arriyam** [comment]
Note: WebServices unit test file is not complete and will be done in a future PR.

The ticket is called: [Pokemon-0013] Unit Test - Add Unit Tests to WebServices

**Review by judarlima — APPROVED** [review]

**Pull request merged**
